test/test_convert.py

```diff
@@ -64,6 +64,8 @@ def assertNoFileTag(self, path, tag):
                                 .format(path, tag))
 
 
+@unittest.skipIf('SKIP_SLOW_TESTS' in os.environ,
+                 'Skipping because test is slow')
```

Comment:
Maybe it would be helpful to wrap this up in some reusable decorator, like `@slow_test`?